knife4j/knife4j-openapi3-jakarta-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.xiaoymin</groupId>
+        <artifactId>knife4j</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>knife4j-openapi3-jakarta-spring-boot-starter</artifactId>
+
+    <properties>
+        <knife4j-java.version>17</knife4j-java.version>
```

Comment:
- This directly requires the entire project to compile under JDK 17, but apart from this module, the other modules produce JDK 8. Since there are no unit tests anywhere, it is difficult for me to verify whether this breaks anything.